framework/jobs/src/main/java/org/apache/cloudstack/framework/jobs/dao/AsyncJobDao.java

```diff
@@ -24,6 +24,10 @@
 import com.cloud.utils.db.GenericDao;
 
 public interface AsyncJobDao extends GenericDao<AsyncJobVO, Long> {
+
+    @Override
+    AsyncJobVO findById(Long id);
```

Comment:
Why is this necessary?

Reply:
Because it was referenced by interface and not implementation. But as discussed, I will remove the override and use findByIdIncludingRemoved.